lib/vfvalue.h

```diff
@@ -280,27 +299,12 @@ namespace ValueFlow
 
         ErrorPath errorPath;
 
-        ErrorPath debugPath;
+        ErrorPath debugPath; // TODO: make lighter by default
```

Comment:
This is only used when `--debug` is specified. So we waste 16 bytes for having as full type instead of a pointer.

I did hack this out locally and it saves about 3% IR and several operations no longer show up in profiling at all so it appears we just narrowly exceed some threshold for some compiler optimization(s) or allocation pattern.

Reply:
Since these objects are copied we cannot simply introduce a pointer.

I also looked into using a lighter container like `std::forward_list` for `ErrorPath` but that doesn't fly either since we insert a lot at the end of it.